LICENSE.md

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
Oh, good catch! This file got here due to some gotestfmt weirdness, see [4992e9288](https://github.com/pulumi/pulumi-eks/pull/979/commits/4992e9288e2e71c1b5157866185a25805a903a73). Removed the file just now.

I poked around a bit but couldn't see why this would happen here and not in all the other repos where we use gotestfmt.

Reply:
I see you fixed the workflow, but looks like the license file is still in the diff

Reply:
fixed